aequilibrae/paths/linear_approximation.py

```diff
@@ -445,6 +465,9 @@ def execute(self):
             c.results.link_loads /= c.pce
             c.results.total_flows()
 
+        # TODO (Jan 18/4/21): Do we want to blob store path files (by iteration, class, origin, destination) in sqlite?
```

Comment:
Can we blob in Feather inside SQLite in the end?  I don't remember where that conversation went.

Reply:
Worth trying? https://stackoverflow.com/a/58980160/1480643

Reply:
Some comments on the design from of sqlite files with blobs from the creator of SQLite himself
http://sqlite.1065341.n5.nabble.com/Effect-of-blobs-on-performance-tp19559p19571.html

Reply:
as just discussed this is out of scope, we'll make it an enhancement issue